### DISCUSSION TITLE
Custom Sliderslide beatmap sample not playing

### DISCUSSION THREAD

**Discussion by MaxKruse** [ppy/osu#24977] [Q&A] [answered]
As seen on this beatmap: https://osu.ppy.sh/beatmapsets/355573#osu/784791

soft-sliderslide99.wav ( 01:13:417 (1) - ) is not being played on lazer, but it being played on stable.

I havent checked if this is also the case with other samples, i just noticed this one while playing.

Replicate: 

1. Change settings to use beatmap hitsounds
2. Pick any skin
3. Forward to the object in question
4. Sample isnt played

**Comment by peppy** [comment, marked as answer]
Thanks for the report. I've written this up at https://github.com/ppy/osu/issues/24997. Unfortunately it's not a simple one to fix so we'll track it as a lower priority issue.
